plugins/welcome-plugin/src/welcome-plugin.ts

```diff
@@ -84,7 +84,7 @@ export async function addPanel(context: theia.PluginContext): Promise<void> {
         // And restrict the webview to only loading content from our extension's `media` directory.
         localResourceRoots: [
             theia.Uri.file(path.join(context.extensionPath, 'resources')),
-            theia.Uri.file(path.join(context.extensionPath, 'node_modules/@fortawesome'))
+            theia.Uri.file(path.join(context.extensionPath, 'node_modules/@fontawesome'))
```

Comment:
Why changed it?
There's no `@fontawesome` package in `node_modules`, but `@fortawesome` only.